activestorage/lib/active_storage.rb

```diff
@@ -60,6 +60,7 @@ module ActiveStorage
   mattr_accessor :content_types_allowed_inline,     default: []
 
   mattr_accessor :service_urls_expire_in, default: 5.minutes
+  mattr_accessor :rails_urls_expire_in
```

Comment:
If you have a better name, please advise!

Reply:
Maybe just `urls_expire_in`? Unsure whether that's too generic but definitely not keen on `rails_urls_expire_in`

Reply:
Thanks.
I think `urls_expire_in` would be a better name.

Reply:
@pixeltrix

Changed to `urls_expire_in`.